examples/react-router/yarn.lock

```diff
@@ -8429,11 +8429,14 @@ react-is@^16.6.0, react-is@^16.7.0:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-16.8.4.tgz#90f336a68c3a29a096a3d648ab80e87ec61482a2"
   integrity sha512-PVadd+WaUDOAciICm/J1waJaSvgq+4rHE/K70j0PFqKhkTBsPv/82UGQJNXAngz1fOQLLxI6z1sEDmJDQhCTAA==
 
+<<<<<<< HEAD
 react-is@^16.8.6:
   version "16.8.6"
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-16.8.6.tgz#5bbc1e2d29141c9fbdfed456343fe2bc430a6a16"
   integrity sha512-aUk3bHfZ2bRSVFFbbeVS4i+lNPZr3/WM5jT2J5omUVV1zzcs1nAaf3l51ctA5FFvCRbhrH0bdAsRRQddFJZPtA==
 
+=======
+>>>>>>> fix(compat): upgrade React Router example lifecycles
```

Comment:
We forgot to run `yarn` to fix the conflict inside the example.

Reply:
I fixed it in the `fix-life-cycles` branch in 373f9bc7d6f328fdbd4ac6f49b7e2bdb413dc0e0.